runtime/include/tasks/qthreads/tasks-qthreads.h

```diff
@@ -117,6 +117,16 @@ extern
 volatile int chpl_qthread_done_initializing;
 
 typedef struct {
+  //chpl_bool serial_state;
```

Comment:
I'd still like to double-check. Do you think it's worth moving lineno and filename to the common bundle area, even though all tasking layers do not need them?